ansible/roles/operator-pipeline/templates/openshift/tasks/publish-to-ocp-registry.yml

```diff
@@ -27,6 +27,10 @@ spec:
         #! /usr/bin/env bash
```

Comment:
According to the param description, this task seems to be publishing the index image rather than the bundle image. Was that intended or are the descriptions incorrect?